backend/src/controllers/v2/workspaceController.ts

```diff
@@ -467,4 +467,42 @@ export const deleteWorkspaceMembership = async (req: Request, res: Response) =>
 	return res.status(200).send({
 		membership
 	});
-}
\ No newline at end of file
+}
+
+/**
+ * Change autoCapitilzation Rule of workspace
+ * @param req
+ * @param res
+ * @returns
+ */
+export const toggleAutoCapitilzation = async (req: Request, res: Response) => {
```

Comment:
Can you change this to `toggleAutoCapitalization`? - Sorry a little nit-picky about spelling haha.